text/0000-utilities-profile.md

```diff
@@ -0,0 +1,42 @@
+- Start Date: 2015-12-03
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes we move `extend`, `select`, and `option` from the base
+specification to its own profile.
+
+Note that I'm calling this a "profile" versus a "namespace." This is pending
+this [RFC](https://github.com/refractproject/rfcs/pull/30). If we do not go the
+route proposed there, we can refer to this as a Utility Namespace.
+
+# Motivation
+
+The main motivation is to make the base specification as simple as possible. In
+ever implementation we've created for consuming Refract, such as from the Parse
+Result or API Description results, we have not needed to use any of these
```

Comment:
Note I mention here that we have no needed the values in "consuming" because these elements do not show up. We also have not needed them in the Fury adapter. So while we do need them in this particular namespace, my point is that:

1. You could remove them from the base spec
1. Build a library that conforms to the base spec
1. Still be able to build/consume the structures we're using

The point was to move these out of the spec so that a library conforming to Refract would be smaller, yet still be able to get the job done.